pyhap/characteristic.py

```diff
@@ -4,6 +4,10 @@
 A Characteristic is the smallest unit of the smart home, e.g.
 a temperature measuring or a device status.
 """
+import logging
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
I know this is pythonic and all that but using `logger` instead is also fairly common and easier to write. What do you think?

Reply:
That's what I would have suggested as well. I will change it to `logger`.